labman/gui/handlers/process_handlers/test/test_pooling_process.py

```diff
@@ -0,0 +1,37 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2017-, labman development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from unittest import main
+
+from tornado.escape import json_decode, json_encode
+
+from labman.gui.testing import TestHandlerBase
+
+
+class TestPoolingProcessHandlers(TestHandlerBase):
```

Comment:
would it be possible to add in failure mode tests?

Reply:
Added where possible